### DISCUSSION TITLE
/var/lib/rancher/k3s/data/xxxxxx is not created

### DISCUSSION THREAD

**Discussion by thanosz** [k3s-io/k3s#9986] [General]
Hello, I am having problems running k3s. My understanding is that `/usr/local/bin/k3s` will download all necessary binary files under `/var/lib/rancher/k3s/data/<random-id>/{bin,etc}`. This is not happening in my case and I am trying to figure out why. Its probably failing due to network restrictions but I cannot understand from where k3s will try to download the files.

The installation succeds, i.e. `curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--disable=traefik" sh -s -` downloads the k3s binary and sets-up everything but the k3s service is failing. Any ideas ? 
```
k3s check-config
INFO[0000] Acquiring lock file /var/lib/rancher/k3s/data/.lock
INFO[0000] Preparing data dir /var/lib/rancher/k3s/data/3fcd4fcf3ae2ba4d577d4ee08ad7092538cd7a7f0da701efa2a8807d44a25f66
ERRO[0001] Hash for file ip expected to be 3c89e5664a7801deab2a5f30fea4d42b6ec7b7ce46389d53024d38de1bb8a526 (fail)
ERRO[0001] Hash for file aux/ebtables-legacy expected to be 86c4da0fabec5f809ffbe725b1a307fa11b0a562d92d4dfe55355a9a9dead221 (fail)
ERRO[0001] Hash for file aux/ebtablesd expected to be c4af716f77714aa5dd16f570a76d4fc890cbcf17d1f7c7b1bed4aff2a67a127e (fail)
ERRO[0001] Hash for file losetup expected to be 8844aae6c777eda05ec672e46c00508cffefc70672911f108bd890506b77290b (fail)
ERRO[0001] Hash for file aux/xtables-legacy-multi expected to be 8cc834717af150b879f9cb9892236fd0fd4695d54bc50fc82a8d31f66a1bfa84 (fail)
ERRO[0001] Unable to verify sums: failed 5 hash verifications
FATA[0001] extracting data: failed to verify directory /var/lib/rancher/k3s/data/3fcd4fcf3ae2ba4d577d4ee08ad7092538cd7a7f0da701efa2a8807d44a25f66-tmp/bin
```

```
ls -la /var/lib/rancher/k3s/data/
total 8
drwxr-xr-x 2 root root 4096 Apr 20 15:21 .
drwxr-xr-x 3 root root 4096 Apr 20 15:09 ..
-rw------- 1 root root    0 Apr 20 15:09 .lock
```

**Comment by thanosz** [comment]
So I have done a little more digging, it seems that I am encountering https://github.com/k3s-io/k3s/issues/9738

**Reply by brandond** [reply]
Sounds like you should disable your AV or at least exclude the k3s dirs from scanning.

**Reply by brandond** [reply]
Please do also open an issue with your AV vendor, complaints from customers are the only thing that will force them to get their act together.
